starter-code/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>revera</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="./css/bootstrap-3.3.7-dist" type="text/css" rel="stylesheet">
```

Comment:
when you link bootstrap from the bootstrap website, you don't have to download it.